importer/views.py

```
@@ -137,65 +137,69 @@ def get(self, request):
         # Instantiate search helper
         search_helper = SearchTool(
-            filename=media_dir, title=title, author=author, keywords=keywords)
+            filename=media_dir, title=title, author=author, keywords=keywords
+        )
 
         # Call search API
         results = self.call_search_api(search_helper)
 
         # Write search result status to log
         if not results:
             logger.warn(
```

Ruff (0.15.0)

[warning] Logging statement uses `warn` instead of `warning`

Convert to `warning`

(G010)